### DISCUSSION TITLE
Steps Component custom numbering

### DISCUSSION THREAD

**Discussion by mankiiii** [primefaces/community#184] [PrimeReact] [answered]
Hello, so in my project im interested in changing the numbers at the top that show on what number step u are. For example my form has 14 steps. I want to only show 10 at the top of my screen not 14. So when u are on step 3 it will show steps 3 to 12 and so on. Is there any way to do this ?

The issue is about the PrimeReact Steps component

**Comment by buraksaglam089** [comment, marked as answer]
Hello,


Currently, we don't have a prop exactly as you want it, but it could potentially be introduced as a new feature in the future. You can use PassThrough options to modify your step component as you want.
